.github/workflows/continuous-integration.yml

```diff
@@ -79,6 +79,19 @@ jobs:
     runs-on: ubuntu-latest
     if: "!contains(needs.pre_ci.outputs.commit_message, '[skip ci]')"
     needs: pre_ci
+    services:
+      postgres:
+        image: postgres:12
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: skyra-nodejs-test
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
```

Comment:
Needs to have Grpc added, so this PR depends on the `feat/docker-dotnet` branch.